test/predis/integration_test.clj

```diff
@@ -126,6 +126,18 @@
       (is (= (r/get mock-client k)
              (r/get carmine-client k))))))
 
+(defspec test-getrange
+  200
```

Comment:
Would dial back to 10 for consistency, I'm probably going to bump up the default spec count later